app/(afterLogin)/(okrplanning)/okr/_components/keyresultForm/currencyForm/index.tsx

```
@@ -261,10 +261,15 @@ const CurrencyForm: React.FC<OKRFormProps> = ({
                   { required: true, message: 'Please enter the target value' },
                   {
-                    validator: (rule, value) =>
-                      value > 0
-                        ? Promise.resolve()
-                        : Promise.reject('Target must be greater than 0'),
+                    validator: (rule, value) => {
+                      if (value <= 0) {
+                        return Promise.reject('Target must be greater than 0');
+                      }
+                      if (value <= keyItem.initialValue) {
+                        return Promise.reject('Target must be greater than initial value');
```

ESLint

[error] Replace `'Target·must·be·greater·than·initial·value'` with `⏎··························'Target·must·be·greater·than·initial·value',⏎························`

(prettier/prettier)